### PR TITLE
Version history

### PR DESCRIPTION
**Pull request by jburel**
Add section for 5.3.0-m5 and cherry-pick commits for 5.2.6 that did not get backported.
cc @hflynn

### CODE REVIEW

**Comment by mtbc** [comment]
Initial capitals and final periods seem inconsistent among the bulleted lists.

**Comment by jburel** [comment]
@hflynn: added sections for other intermediate milestones

**Comment by hflynn** [comment]
Previous entries have been past tense ('fixed' 'improved' etc) but I'm not too fussed here given it'll be re-written for the public release. I'll hunt thro the cards and see if I spot anything we've missed. Would be good to point at this for the OMERO status blog post once published.

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,93 @@
...
+
+5.3.0-m2 (April 2016)
+---------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+- added support for ROI folder at scale. Used IDR as a concrete example
```
Comment (hflynn) [inline history.txt]: https://trello.com/c/LW8l7U8x/20-shape-attribute-renaming-cleanup is labelled for m2 too - worth adding something about this?

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,93 @@
...
+
+-  added DB patches including support new options in rendering
+-  added support for LUT
+-  deprecated windows
+-  reorganized API documentation
+-  enabled plate download
+-  allowed customization of CSRF cookie
+-  fixed ROI related bug
+-  performed major code cleanup
+-  fixed cleanse
```
Comment (hflynn) [inline history.txt:52]: Add deprecated OmeroPopo https://trello.com/c/0dkqUU9f/51-deprecate-omeropopo-py ?

File: history.txt [hunk 3]
```diff
@@ -1,6 +1,93 @@
...
+
+5.3.0-m3 (August 2016)
+----------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+-  added DB patches including support new options in rendering
```
Comment (hflynn) [inline history.txt]: "...support **for** new..."

File: history.txt [hunk 4]
```diff
@@ -1,6 +1,93 @@
...
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+-  database patches to add support for new administration roles
+-  removed restriction on name length
+-  added support for enumeration changes
+-  added new Grid View to both UI clients
+-  introduced the new Web API
```
Comment (hflynn) [inline history.txt:16]: Add making web apps pip installable too?
Open With?

**Review by hflynn — COMMENTED** [review]
Few suggestions and a very minor word addition nitpick

File: history.txt (hunk 2 above)
Comment (jburel) [reply history.txt:52]: I put that under major code clean up since nobody's know what OmeroPopo is

Comment (hflynn) [reply history.txt:52]: Fair enough!

Comment (jburel) [reply history.txt:52]: went for a generic entry in the new commit

File: history.txt [hunk 5]
```diff
@@ -16,7 +16,10 @@ Changes include:
 -  introduced the new Web API
 -  reactivated codomain support in the rendering Engine
 -  added support for Ice 3.6.3
--  fixed graph issues when performing chown or moving between groups.
+-  fixed graph issues when performing chown or moving between groups
+-  made web applications installable from PyPI
+-  added "Option With" feature to OMERO.web
```
Comment (hflynn) [inline history.txt]: Option -> Open

File: history.txt [hunk 6]
```diff
@@ -56,7 +60,8 @@ updating code before the full public release of 5.3.0. Use at your own risk**.
 
 Changes include:
 
-- added support for ROI folder at scale. Used IDR as a concrete example
+-  added support for ROI folder at scale. Used IDR as a concrete example
+-  cleaned up shape attributes names
```
Comment (hflynn) [inline history.txt]: "attribute" singular since followed by name**s**

**Review by hflynn — COMMENTED** [review]
Couple of nitpicks but otherwise looks good to me

**Review by hflynn — APPROVED** [review]

**Comment by jburel** [comment]
by-passing travis. No code change

**Pull request merged**